### PR TITLE
[3] 2022/section headers

### PR DESCRIPTION
**Pull request by zastrow**
This work setup up the section headers with a title, number, description and diamond shape.

[Figma Comp](https://www.figma.com/file/VcIuWHbkjExLZ8SNQZipAo/2022-DS-Survey?node-id=2713%3A945)

CleanShot 2022-06-06 at 09 32 37@2x

## What needs checked:
- [x] Install and run the project using the README instructions
- [x] Sass code looks good
- [x] React code looks good

## What does not need checked
- Design accuracy. The design is still a work in progress with changes coming in the browser as needed. It is more important to check for legibility and aesthetic feel when it comes to design.
- Small screen. There are known small screen bugs, a following branch will address responsive issues.

### CODE REVIEW

File: src/scss/2022/components/_section-header.scss [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+  &::after {
+    content: '';
+    display: block;
+    height: min(40vmin, 26rem);
+    width: min(40vmin, 26rem);
+    background: radial-gradient(circle at 200% 50%,
+                                hsla(var(--color-pink-1-val), 1) 0%,
+                                hsla(var(--color-pink-2-val), 1) 60%,
+                                hsla(var(--color-black-val), 0) 95%)
+                                no-repeat;
```
Comment (bengoshow) [inline src/scss/2022/components/_section-header.scss:19]: This is so cool. Elegant solution.

File: src/scss/2022/components/_section-header.scss [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+    line-height: 1;
+    position: relative;
+    padding-top: 3.5rem;
+    padding-left: 5rem;
+
+    &::before {
+      pointer-events: none;
+      font-family: var(--font-text);
+      @include typography.number('small');
+      content: '0' attr(data-number);
```
Comment (bengoshow) [inline src/scss/2022/components/_section-header.scss:40]: Also very cool, accessing the data attribute. 🎉

**Review by bengoshow — APPROVED** [review]
Everything here looks great, @zastrow - nice work!
goldblum_very_good

File: src/scss/2022/components/_section-header.scss [hunk 3]
```diff
@@ -0,0 +1,56 @@
...
+      position: absolute;
+      top: 0;
+      left: 0;
+      display: inline-flex;
+      place-items: start;
+      height: 8rem;
+      width: 6rem;
+      color: var(--color-gray);
+      background: svg('<svg width="290" height="439"><path d="m287 2.327-284 435" stroke="#{color(dark)}" stroke-width="5"/></svg>') no-repeat center / contain;
```
Comment (lunarfusion) [inline src/scss/2022/components/_section-header.scss:49]: Maximizing the potential of the pseudo-element with content AND a background! Really nice way to get those two elements in.

File: src/scss/2022/components/_section-header.scss (hunk 1 above)
Comment (lunarfusion) [reply src/scss/2022/components/_section-header.scss:19]: Another cheer for this - I appreciate how the blend modes interact as the diamond passes over the circle.

**Comment by lunarfusion** [comment]
Looked over scss, checked visually in Chrome and Firefox.

**Pull request merged**